loadtest/user/userentity/actions.go

```diff
@@ -163,3 +163,25 @@ func (ue *UserEntity) GetChannelStats(channelId string) error {
 	}
 	return err
 }
+
+func (ue *UserEntity) GetTeams() ([]string, error) {
+	user, err := ue.store.User()
+	if user == nil || err != nil {
+		return nil, errors.New("user was not initialized")
+	}
```

Comment:
Here we should use `ue.getUserFromStore()` like in all other cases.

Reply:
Oops! On it :)